shenyu-common/src/main/java/org/apache/shenyu/common/exception/ThrowingFunction.java

```diff
@@ -15,32 +15,37 @@
  * limitations under the License.
  */
 
-package org.apache.shenyu.plugin.jwt.lambda;
+package org.apache.shenyu.common.exception;
 
 /**
- * Exception wrap.
+ * Throwing wrap.
  */
 @FunctionalInterface
-public interface ThrowingFunction {
+public interface ThrowingFunction<T> {
+
 
     /**
      * apply function.
      *
-     * @return object
+     * @return T
+     * @throws Exception any error
      */
-    Object apply();
+    T apply() throws Exception;
 
     /**
      * operation function.
      *
-     * @param function accept function
-     * @return object
+     * @param function apply function.
+     * @param <T> t
+     * @return T
      */
-    static Object wrap(ThrowingFunction function) {
+    static <T> T wrap(ThrowingFunction<T> function) {
         try {
             return function.apply();
         } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
I don't think `e.printStackTrace();` is a good idea.

Reply:
I will resubmit a code, please review again, thank you.